scivision/catalog/data/models.json

```diff
@@ -136,6 +136,17 @@
             "labels_required":false,
             "institution":["alan-turing-institute", "huggingface"],
             "tags": []
+        },
+        {
+            "name":"detectreeRGB-forest",
+            "tasks":["object-detection","segmentation"],
+            "url":"https://github.com/shmh40/detectreeRGB-treecrown-scivision",
+            "pkg_url":"git+https://github.com/shmh40/detectreeRGB-treecrown-scivision.git@main",
+            "format":"image",
+            "pretrained":true,
+            "labels_required":false,
+            "institution":["cambridge-university"],
+            "tags": ["object-detection","segmentation", "2D", "image"]
```

Comment:
I realise this may be the case with other catalog entries too, but probably no point having `tags` which are the same values as `tasks` and also I guess the tag "image" seems unnecessary too :)